View Controllers/FeedViewController.m

```diff
@@ -42,6 +49,7 @@ - (void)fetchEvents {
     // get events from current user
     [query whereKey:@"user" equalTo:currentUser];
     [query findObjectsInBackgroundWithBlock:^(NSArray * _Nullable objects, NSError * _Nullable error) {
+        self.events = [NSArray new];
```

Comment:
is this necessary?

Reply:
That's there so that when you refresh it clears the list we already have of events